lib/projections/qsc.js

```diff
@@ -0,0 +1,368 @@
+// QSC projection rewritten from the original PROJ4
+// https://github.com/OSGeo/proj.4/blob/master/src/PJ_qsc.c
+
+import {EPSLN, TWO_PI, SPI, HALF_PI, FORTPI} from '../constants/values';
+
+export function init() {
+
+  this.x0 = this.x0 || 0;
+  this.y0 = this.y0 || 0;
+  this.lat0 = this.lat0 || 0;
+  this.long0 = this.long0 || 0;
+  this.lat_ts = this.lat_ts || 0;
+  this.title = this.title || "Quadrilateralized Spherical Cube";
+
+  this.face_enum = {
+    FRONT: 1,
+    RIGHT: 2,
+    BACK: 3,
+    LEFT: 4,
+    TOP: 5,
+    BOTTOM: 6
+  };
+
+  this.area_enum = {
```

Comment:
this should probably be a constant defined outside of the instance instead of defined everytime